src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  health = 100;
```

Comment:
```suggestion
  constructor(name, health = DEFAULT_HEALTH) {
    this.name = name;
    this.health = health;
    
    Animal.alive.push(this);
  }
```
- You should accept health in case it will be different, just by default it should be 100
- 100 and 50 are magic values, create constants for them